themes-default/slim/src/store/modules/shows.js

```diff
@@ -81,20 +81,32 @@ const actions = {
      * @param {ShowParameteres[]} shows - Shows to get. If not provided, gets the first 10k shows.
      * @returns {(undefined|Promise)} undefined if `shows` was provided or the API response if not.
      */
-    getShows(context, shows) {
+    async getShows(context, shows) {
```

Comment:
`async` + `while` is likely causing this. 🤔

Reply:
Then I'd still have a loop. A for loop instead of while.

Btw I got the infinite loop when I disabled the detailed flag in the show.vue component?

Reply:
A while loop can cause issues inside of an async function. 

Have a look at the answer from this.

https://stackoverflow.com/questions/49155139/while-loop-inside-async-await